Updated textures, items, and liquid visibility

Lightning323 opened the pull request:
Sorry for the delay. This is a pretty big one. I haven't gotten around to updating everything just yet, so I will be making future follow up PRs for tweaks or if I find that I missed something.


In a nutshell:
- Updated the textures of many items and blocks for a full refresh
     - The goal was to make everything blend in seamlessly with create (make it actually feel like a create mod)
     - Most of the blocks now have an "encased" texture to better blend in with create
- Improved item textures
- Improved some planet textures
**- Improved visibility of all liquids (big deal)**
- Changed a few of the blocks to brass (engraver, oxygen filler, etc.) that are pre-space anyway
- Changed the combustion chamber from copper to brass
- Added durable fabric to space suits
- Added new items for improving space travel progression, like a rocket combustion chamber (item), oxygen separator (item) and hardened precision mechanism, (made from iron cogwheels)
- Added titanium, A new alloy made form iron and sand
- Changed some recipes for items and blocks for better balance, progression, and game mechanics

Lightning323 committed:
Updated textures, items, and liquid visibility

Updated textures, added a few items
- Updated the textures of many items and blocks for a full refresh
- Improved visibility of all liquids,
- Changed a few of the blocks to brass (engraver, oxygen filler, etc.) that are pre-space anyway
- Changed the combustion chamber from copper to brass
- Added durable fabric to space suits
- Added hardened precision mechanism, made from iron cogwheels
- Changed a few recipes and added items for better balance and game mechanics

Redstonneur1256 committed:
Merge branch '1.20.1/dev' into redux/1.20/updatedTexturesItems

Redstonneur1256 commented:
Some problems that need to be fixed before merging:
- The compressed sand bar either requires 9 sandstone blocks (equivalent to 36 sand) or 16 sand blocks making the recipes inconsistent (unless this is a deliberate decision?)
- Broken iron space suit helmet crack isn't visible enough making it inconsitent with the item texture
- Broken iron space suit boots and other parts are pure black (missed spot?)
- Titanium block cannot be crafted from ingots or crafted back into ingots
- Mixed titanium/iron blocks, if you want to fully replace iron related stuff with the titanium one please explain.
- The circuit engraver laser's texture could use a simple downwards animation
- The animation for the oxygen detector and interplanetary navigator just doesn't feel right, for the interplanetary navigator maybe a fade in/out with a light spinning on the big disc? as for the oxygen detector the no oxygen texture looks blurry with the animation, maybe a face similar to x_x could match better, as for the animation of with/without oxygen a simple pulse could be better (or maybe no animation like the original one)

Lightning323 committed:
texture and recipe fixes

Lightning323 committed:
texture and recipe fixes

Lightning323 committed:
Merge branch 'redux/1.20/updatedTexturesItems' of https://github.com/zipCoder933/Northstar-True-North into redux/1.20/updatedTexturesItems

Lightning323 committed:
Fixed space suit textures

Lightning323 committed:
Fixed everything (hopefully)

- Finally finished interplanetary navigator
- updated rocket controls, and station

Lightning323 committed:
Fixed everything (hopefully)

- Finally finished interplanetary navigator
- updated rocket controls, and station

Lightning323 committed:
Merge branch 'redux/1.20/updatedTexturesItems' of https://github.com/zipCoder933/Northstar-True-North into redux/1.20/updatedTexturesItems

Lightning323 committed:
Revert "Merge branch 'redux/1.20/updatedTexturesItems' of https://github.com/zipCoder933/Northstar-True-North into redux/1.20/updatedTexturesItems"

This reverts commit 8530b4dbdc1810617c530a8d33c8f735ea06e5ef, reversing
changes made to 1967f765ea51d7895fc3cc819489e1f723d6035d.

zipCoder933 commented:
I think I got everything...
Feel free to merge it now.

Lightning323 committed:
fixed rocket control item models

Lightning323 committed:
updated fixed item models
